src/fondant/core/schema.py

```diff
@@ -181,3 +203,16 @@ def validate_partition_size(arg_value):
         )
         raise InvalidTypeSchema(msg)
     return arg_value
+
+
+@dataclass
+class ManifestField(Field):
```

Comment:
Do we need to have a separate `ManifestField`, or should we update `Field` to optionally accept a location? I would not give it a `base_path` property. Either let it only hold a relative location, and require the manifest to resolve the absolute path. Or just let the field hold the absolute path.